From: Antonio Valentino <antonio.valentino@tiscali.it>
Date: Mon, 24 Jun 2024 18:23:27 +0000
Subject: Fix test_max_threads_unset

Forwarded: https://github.com/pydata/numexpr/pull/491
---
 numexpr/tests/test_numexpr.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/numexpr/tests/test_numexpr.py b/numexpr/tests/test_numexpr.py
index 15fb2c0..c69e61d 100644
--- a/numexpr/tests/test_numexpr.py
+++ b/numexpr/tests/test_numexpr.py
@@ -1129,7 +1129,7 @@ class test_threading_config(TestCase):
                 "if 'NUMEXPR_MAX_THREADS' in os.environ: os.environ.pop('NUMEXPR_MAX_THREADS')",
                 "if 'OMP_NUM_THREADS' in os.environ: os.environ.pop('OMP_NUM_THREADS')",
                 "import numexpr",
-                "assert(numexpr.nthreads <= 8)",
+                f"assert(numexpr.nthreads <= {MAX_THREADS})",
                 "exit(0)"])
         subprocess.check_call([sys.executable, '-c', script])
 
